use-cases/model-fine-tuning-pipeline/data-processing/ray/README.md

```diff
@@ -57,6 +59,7 @@ The data processing step takes approximately 18-20 minutes.
   --project ${MLP_PROJECT_ID} \
   --substitutions _DESTINATION=${MLP_DATA_PROCESSING_IMAGE}
   cd ..
+  rm -rf src/datapreprocessing
```

Comment:
Why are you removing the modules?

Reply:
just to avoid having 3 copies of the modules lying around in three different directories. It is not required to remove the modules but I thought it would be cleaner.